styles/main.scss

```diff
@@ -46,8 +76,12 @@ body {
     cursor: pointer;
     display: flex;
     padding: 5px 14px;
+    white-space: nowrap;
+    text-overflow: ellipsis;
+    overflow: hidden;
+
 
-    &:hover, &.selected {
```

Comment:
Do we really want to remove highlighting on hover via CSS? Implementing via JS gets annoying, although with `mouseenter` and `mouseleave` it's not as bad is it used to be.